src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlCheck.java

```diff
@@ -80,22 +80,22 @@
  * </li>
  * <li>
  * If there is matching allow/disallow rule inside the current file/subpackage
- * then the Check returns the first "allowed" or "disallowed" message.
+ * then the Check returns the first "allowed" or "forbidden" message.
```

Comment:
We might need to make note section to mention about synonyms, to let users migrate easily, or find easily why configs in web are different on this.